src/elements/content-sidebar/activity-feed/annotation-thread/__tests__/useAnnotationThread.test.js

```diff
@@ -235,5 +236,13 @@ describe('src/elements/content-sidebar/activity-feed/useAnnotationThread', () =>
 
             expect(result.current.annotation).toEqual({ ...updatedAnnotation, isPending: false });
         });
+
+        test('Should emit annotation active change event when used', () => {
+            const event = useAnnotatorEvents();
+
+            getHook();
+
+            expect(event.emitAnnotationActiveChangeEvent).toHaveBeenCalled();
```

Comment:
Is this desirable if nothing has actually changed from one hook call to the next?

Reply:
Actually we call tested hook (`useAnnotationThread`) only once here. So it's desirable for the event to be called once. I'll try to improve the test to make it more clear.